force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +214,24 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String result = '';
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
+         //for () { 
+        Integer repeatCount = 3;
+        String inputString = 'Hello World!';
+            for (Integer i = 0; i < repeatCount; i++) {
+                result += inputString;
+                if (i < repeatCount - 1) {
+                    result += '; ';
+                }
+            }
+        System.debug(result);           
```

Comment:
Don't hard code the value of repeatCount that is meant to be an argument that the caller provides.  The other logic looks correct. Check out the String method removeEnd();